apps/condo/domains/acquiring/schema/RegisterMultiPaymentService.js

```diff
@@ -479,6 +498,186 @@ const RegisterMultiPaymentService = new GQLCustomSchema('RegisterMultiPaymentSer
                 }
             },
         },
+        {
+            access: access.canRegisterMultiPayment,
+            schema: 'registerMultiPaymentForOneReceipt(data: RegisterMultiPaymentForOneReceiptInput!): RegisterMultiPaymentOutput',
```

Comment:
it's better to use `createservice service.RegisterMultiPaymentForOneReceiptService`